frontends/web/src/routes/new-settings/components/singledropdown/singledropdown.module.css

```diff
@@ -0,0 +1,50 @@
+.dropdown {
+    appearance: none;
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24' fill='none' stroke='currentColor' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-labelledby='chevron down' color='%23777'%3E%3Cdefs/%3E%3Cpolyline points='6 10 12 16 18 10'/%3E%3C/svg%3E%0A");
+    background-repeat: no-repeat;
+    font-weight: 400;
+    height: calc(var(--space-quarter) * 3);
+    padding: 0 calc(var(--space-quarter) + var(--space-eight));
+}
+
+.select {
+    width: 280px;
+}
+
+.select :global(.react-select__menu) {
+    background-color: var(--background-secondary);
+}
+
+.select :global(.react-select__option) {
+    background-color: var(--background-secondary);
+}
+
+.select :global(.react-select__option):hover {
+    background-color: var(--background-custom-select-hover);
+}
+
+.select :global(.react-select__option--is-selected),
+.select :global(.react-select__option--is-selected):hover {
+    background-color: var(--background-custom-select-selected);
+}
+
+.select :global(.react-select__control) {
+    background-color: var(--background-secondary);
+    border: 2px solid var(--color-custom-select-border);
+    border-radius: 0;
+    padding: 0 var(--space-eight);
+}
+
+.select :global(.react-select__control):hover {
+    cursor: pointer;
+}
+
+.valueContainer {
+    font-size: 16px;
```

Comment:
isn't there a CSS variable that you can use for the font-size here?

Reply:
Actually, we do. It's

```css
   /* wizard */
    --size-wizard-text: 16px;
```

sounds a bit specific (the naming) so I refrain from using it. Should we use this?

Reply:
I think 14px is good. That's the same size we have for the current currencies.

Reply:
Looking at variables.css I think we should cleanup this up globally, not in this PR.

14px is ok, normal text input fields seem to also use 14px.

Reply:
> Looking at variables.css I think we should cleanup this up globally, not in this PR.

Yeah, not going to modify already existing variables here thus going with `--size-default`  now (14px).  Had the impression of the text in the design being 16px not 14px..  but clearly 14px it is:)

Reply:
> Had the impression of the text in the design being 16px not 14px.. but clearly 14px it is:)

yeah me too, I think in view.modules.css it is bigger (18px). Generally we should move to bigger font one day (soonish).